### DISCUSSION TITLE
Option to skip only cloud cache

### DISCUSSION THREAD

**Discussion by skoppell** [nrwl/nx#26825] [Feature Requests]
Hi, 

Currently we have `--skip-nx-cahe` which ignores both local and cloud cache.  Apart from that,  having an option to skip only the cloud-cache will be good addition to have. 

We have cloud-cache enabled for our project. (Nx+Angular).  In our regular pull-request pipelines,  I have configured the cache inputs such that the affected command results in minimal projects and which leads to test target being run only for those minimal set of projects.   however, to make sure all the test in the repo are in good state,  I want to configure a scheduled pipeline to run the test target on all projects once in a day.  So, i configured it using `run-many --all` with `--skip-nx-cache`.   

Our monorepo has more than 200 libraries so if the job fails due to a timeout, i would like to re-run the job. but that would result in running the tests on all project again as it uses `--skip-nx-cache` argument.  If there was a option to skip only cloud cache,  It would help the subsequent run to pick the results from local cache and run the task only for the projects failed in the previous run.  (as the job will be run on the same CI node in the subsequent run)

Or Do we already have a way to achieve what i am looking for ?

**Comment by skoppell** [comment]
Can someone please advice ?

**Comment by Tungsten78** [comment]
This is supported  https://nx.dev/recipes/running-tasks/skipping-cache#skip-remote-caching-from-nx-cloud
